### PR TITLE
Switch to dashboard v2 and remove heapster

### PR DESCRIPTION
**Pull request by linki**
Update to v2 of the [Kubernetes dashboard](https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/)

In v2 the dashboard doesn't rely on heapster which isn't compatible with https://github.com/zalando-incubator/kubernetes-on-aws/pull/2875.

The dashboard changed some auth related things. To get the original behaviour back we serve on the insecure port.

Users can to `kubectl proxy` and then open the dashboard in their browsers at:

http://localhost:8001/api/v1/namespaces/kube-system/services/kubernetes-dashboard/proxy/#/overview?namespace=default

or via `kubectl -n kube-system port-forward svc/kubernetes-dashboard 9090:80` at:

http://localhost:9090/#/overview?namespace=default

### CODE REVIEW

File: cluster/manifests/dashboard/rbac.yaml [hunk 1]
```diff
@@ -3,11 +3,92 @@ kind: ServiceAccount
...
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubernetes-dashboard
+  labels:
+    application: kubernetes-dashboard
+rules:
+# Allow Metrics Scraper to get metrics from the Metrics server
```
Comment (aermakov-zalando) [inline cluster/manifests/dashboard/rbac.yaml]: That looks like an extremely limited role. How will the users be able to interact with deployments, services, etc?

**Comment by linki** [comment]
Double check: `kubectl top pods`

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**